docker-compose.yml

```diff
@@ -79,16 +98,17 @@ services:
       #- MAIL_FROM_NAME=
       #- MAIL_FROM_ADDRESS=
       #- TRUSTED_PROXIES=
-      #- SKIP_PERMISSIONS_CHECKS
+      - SKIP_PERMISSIONS_CHECKS=${SKIP_PERMISSIONS_CHECKS:-false}
       - STARTUP_DELAY=30
       #- ADMIN_USER=admin
       #- ADMIN_PASSWORD=<ADMIN_PASSWORD>
       #- ADMIN_PASSWORD_FILE=<filename>
-      ### Unused in Lychee
-      #- REDIS_HOST=127.0.0.1
-      #- REDIS_PASSWORD=null
-      #- REDIS_PASSWORD_FILE=<filename>
-      #- REDIS_PORT=6379
+      - CACHE_DRIVER=${CACHE_DRIVER:-redis}
+      - REDIS_URL=redis://${REDIS_USERNAME:-default}:${REDIS_PASSWORD:-}@${REDIS_HOST:-lychee_redis}:${REDIS_PORT:-6379}
```

Comment:
This one's not in Lychee's example. Would it be better constructed on that side?

Reply:
Indeed, the are a bit hidden but still visible here:
https://github.com/LycheeOrg/Lychee/blob/master/config/database.php#L181

Reply:
done: https://github.com/LycheeOrg/Lychee/pull/3003